packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -2080,6 +2081,150 @@ void main() {
     scrollable.controller.jumpTo(0);
     await verifyVisibility(false, #right, false, #right);
   });
+
+  testWidgets('iOS text selection handle visibility', (WidgetTester tester) async {
+    debugDefaultTargetPlatformOverride = TargetPlatform.iOS;
+    final GlobalKey<EditableTextState> editableTextKey =
+        GlobalKey<EditableTextState>();
+
+    const String testText = 'XXXXX          XXXXX';
+    final TextEditingController controller = TextEditingController(text: testText);
+
+    final Widget widget = MaterialApp(
```

Comment:
You don't seem to be using this reference later. Maybe just inline this into the pumpWidget?